### PR TITLE
deprecate c8s images, as EOL for c8s is 31th of May 2024

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

**Comment by phracek** [comment]
[test]

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -21,83 +21,55 @@ jobs:
...
+
           - dockerfile: "14/Dockerfile.fedora"
             registry_namespace: "fedora"
             tag: "14"
             quayio_username: "QUAY_IMAGE_FEDORA_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
             image_name: "postgresql-14"
             docker_context: 14
+
           - dockerfile: "15/Dockerfile.fedora"
```
Comment (phracek) [inline .github/workflows/build-and-push.yml:65]: This version is present https://src.fedoraproject.org/rpms/postgresql15

File: .github/workflows/build-and-push.yml [hunk 2]
```diff
@@ -21,83 +21,55 @@ jobs:
...
             registry_namespace: "fedora"
             tag: "15"
             quayio_username: "QUAY_IMAGE_FEDORA_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
             image_name: "postgresql-15"
             docker_context: 15
+
           - dockerfile: "16/Dockerfile.fedora"
```
Comment (phracek) [inline .github/workflows/build-and-push.yml:73]: This version is present https://src.fedoraproject.org/rpms/postgresql16

File: .github/workflows/build-and-push.yml [hunk 3]
```diff
@@ -21,83 +21,55 @@ jobs:
...
+
           - dockerfile: "12/Dockerfile.fedora"
             registry_namespace: "fedora"
             tag: "12"
             quayio_username: "QUAY_IMAGE_FEDORA_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_FEDORA_BUILDER_TOKEN"
             image_name: "postgresql-12"
             docker_context: 12
+
           - dockerfile: "13/Dockerfile.fedora"
```
Comment (phracek) [inline .github/workflows/build-and-push.yml:49]: This can be dropped https://src.fedoraproject.org/rpms/postgresql12

**Review by phracek — CHANGES_REQUESTED** [review]
I think we can drop some Fedora versions. @fila43 What do you think?

File: .github/workflows/build-and-push.yml [hunk 4]
```diff
@@ -21,83 +21,55 @@ jobs:
...
-            docker_context: 13
-          - dockerfile: "15/Dockerfile.c8s"
-            registry_namespace: "sclorg"
-            tag: "c8s"
-            quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
-            quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
-            image_name: "postgresql-15-c8s"
-            docker_context: 15
+
           - dockerfile: "12/Dockerfile.fedora"
```
[the remark this answers is not in the document]
Comment (zmiklank) [reply .github/workflows/build-and-push.yml:41]: I think we should do regular Fedora cleanup again - migrate all images to new Fedoras, and remove unsupported images. There will be more outdated content I think. I suggest doing it in different PR.

File: .github/workflows/build-and-push.yml (hunk 3 above)
Comment (phracek) [reply .github/workflows/build-and-push.yml:49]: Issue is here: https://github.com/sclorg/postgresql-container/issues/570

File: .github/workflows/build-and-push.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (phracek) [reply .github/workflows/build-and-push.yml:41]: Issue is here: https://github.com/sclorg/postgresql-container/issues/570

**Review by phracek — APPROVED** [review]
LGTM.

**Pull request merged**